general/standards/electrical-software-quality-management.rst

```diff
@@ -1,2 +1,22 @@
 Electrical & Software Quality Management Process
 ================================================
+
+PCB designs
+-----------
+
+* Minimum 4 approves for schematics review
+* Minimum 4 approves for PCB layout
+* Minimum 4 approves for routed PCB
+* For each of the 4 reviewers mentioned above, at minimum 2 need to have experience on designing PCB for Waterloo Rocketry team
+* All electrical standard need to be followed, and any exception standard must be explicitly noted
+
+Firmware (Embedded Software)
+----------------------------
+
+* Each PR need at minimum 2 approve to get merged
+* One of the approve have to come from CODEOWNERS (usually project owner)
```

Comment:
A significant number of PRs will likely be coming from the project owner. For smaller boards that might only be one person. I don't think this is practical.

Reply:
maybe this indicates a different issue, that projects shouldn't only have 1 codeowner...? even for small boards, i'd think its always better to have at least 2 people responsible.